src/pages/workspaces/workspace/Data.js

```diff
@@ -566,7 +566,7 @@ const WorkspaceData = _.flow(
   }
 
   const loadWdsSchemaAndUrl = async () => {
-    if (isFeaturePreviewEnabled('workspace-data-service') && !getConfig().isProd && isAzureWorkspace) {
+    if (!getConfig().isProd && isAzureWorkspace) {
```

Comment:
echoing Jordan's comment here, we'll need to remove the `isProd` check at some point. I suggest we make a new ticket and do this in January when everybody is back to work.

Reply:
Sounds good -- made a ticket! https://broadworkbench.atlassian.net/browse/AJ-757